### PR TITLE
[Fix] keras.ops.sort(axis=None) fails on eager tensors instead of flattening and sorting

### PR DESCRIPTION
**Pull request by ChiragSW**
Fixes issue: #22543

### CODE REVIEW

**Comment by ChiragSW** [comment]
Please review, the issue is now fixed.
